maxide.bmx

```diff
@@ -679,6 +685,18 @@ Type TAboutRequester Extends TRequester
 		Return GetProcessOutput(gppPath, "-dumpversion").Split("~n")[0]
 	EndMethod
 
+	Method GetUPX$()
+		Local upxPath:String = BlitzMaxPath() + "/bin/upx"
+		?Win32
+		upxPath :+ ".exe"
+		?
+		If FileType(upxPath) = 1

```

Comment:
You shouldn't use seemingly random numbers when there are consts available. (e.g. FILETYPE_FILE)

When I change file types to an enum, all your code will break...